src/stores/walletStore/walletStore.ts

```diff
@@ -179,12 +179,12 @@ export class WalletStore {
   public async connectWallet(opts: {
     chainId?: string;
     walletProvider: WalletProvider;
-    providerId?: string;
+    rdns?: string;
   }) {
     switch (opts.walletProvider) {
       case WalletProvider.EIP6963: {
-        if (opts.providerId) {
-          await this.connectEIP6963Provider(opts.providerId);
+        if (opts.rdns) {
+          await this.connectEIP6963Provider(opts.rdns);
```

Comment:
When this function gets called in the view layer, it is called with the entire `provider` so no need to make any adjustments there.